README.md

```diff
@@ -1,2 +1,67 @@
-# Capstone-aka-Budget-App
-The BudgetApp is a mobile web application where you can manage your budget: you have a list of transactions associated with a category/many categories, so that you can see how much money you spent and on what item you spent it on
+# BudgetApp
```

Comment:
- [ ] Include a video presentation for your project, these are the requirements
![image](https://user-images.githubusercontent.com/62705947/190194530-5621d6e7-04ac-447e-aade-780066f59933.png)

Reply:
Hi, sorry but vide presentation link is in PR . Can you double check it plz?